src/main/java/org/candlepin/subscriptions/subscription/SubscriptionSyncController.java

```diff
@@ -172,8 +172,7 @@ void syncSubscriptions(String orgId, int offset, int limit) {
     boolean hasMore = subscriptions.size() >= pageSize;
     subscriptions.forEach(this::syncSubscription);
     if (hasMore) {
-      offset = offset + limit;
-      enqueueSubscriptionSync(orgId, offset, limit);
+      enqueueSubscriptionSync(orgId, offset + limit, limit);
```

Comment:
This was changed so that the log statement a couple lines down would become correct. It was reporting that the next page of subscriptions were synced, when in fact it was the previous page that got synced.